rosidl_generator_c/resource/msg__functions.c.em

```diff
@@ -81,7 +81,10 @@ for field in spec.fields:
         # non-array field
         if field.type.is_primitive_type():
             if field.type.type == 'string':
-                lines.append('rosidl_generator_c__String__init(&msg->%s);' % field.name)
+                lines.append('if (!rosidl_generator_c__String__init(&msg->%s)) %s' % (field.name, '{'))
```

Comment:
Nit: please place the `{` in the string itself rather that through a `%s` to stay consistent with what you do [line 106](https://github.com/ros2/rosidl/pull/220/files#diff-633804c447473336f244128adc0b96f3R106)